tofu/util.py

```diff
@@ -86,7 +86,7 @@ def positive_int(value):
     return result
 
 
-def tupleize(num_items, conv=None):
+def tupleize(num_items=None, conv=None):
```

Comment:
I wonder what's the point in calling `tupleize()` without any arguments? Is this the case anywhere?

Reply:
Nope but it will be soon because conv will be float by default. Don't forget that it actually returns a function which _always_ requires one argument.